### PR TITLE
[USER32] ordinals -> @

### PR DESCRIPTION
**Pull request by tkreuzer**
## Purpose

Change hardcoded ordinals to @ to allow adding NT6+ exports.

### CODE REVIEW

**Review by DarkFire01 — APPROVED** [review]

**Comment by whindsaks** [comment]
Kernel32 had ordinal-only exports on Win9x. I don't think User32 nor Gdi32 ever had any so this should be safe.

**Review by whindsaks — APPROVED** [review]

File: win32ss/user/user32/user32.spec [hunk 1]
```diff
@@ -1,752 +1,752 @@
...
+@ stdcall TranslateAcceleratorA(long long ptr)
+@ stdcall TranslateAcceleratorW(long long ptr)
+@ stdcall TranslateMDISysAccel(long ptr)
+@ stdcall TranslateMessage(ptr)
+@ stdcall TranslateMessageEx(ptr long)
+@ stdcall UnhookWinEvent(long) NtUserUnhookWinEvent
+@ stdcall UnhookWindowsHook(long ptr)
+@ stdcall UnhookWindowsHookEx(long) NtUserUnhookWindowsHookEx
+@ stdcall UnionRect(ptr ptr ptr)
+@ stdcall UnloadKeyboardLayout(long)
```
Comment (HBelusca) [inline win32ss/user/user32/user32.spec:708]: The HKL is a handle, so this should stay `ptr`.
```suggestion
@ stdcall UnloadKeyboardLayout(ptr)
```

File: win32ss/user/user32/user32.spec [hunk 2]
```diff
@@ -1,752 +1,752 @@
...
+@ stdcall UnregisterDeviceNotification(long)
+@ stdcall UnregisterHotKey(long long) NtUserUnregisterHotKey
+@ stdcall UnregisterMessagePumpHook()
+@ stdcall UnregisterUserApiHook() NtUserUnregisterUserApiHook
+@ stdcall UpdateLayeredWindow(long long ptr ptr long ptr long ptr long)
+@ stdcall UpdateLayeredWindowIndirect(long ptr)
+@ stdcall UpdatePerUserSystemParameters(long long)
+@ stdcall UpdateWindow(long)
+@ stdcall User32InitializeImmEntryTable(ptr)
+@ stdcall UserClientDllInitialize(long long ptr) DllMain
```
Comment (HBelusca) [inline win32ss/user/user32/user32.spec:722]: The first parameter of UserClientDllInitialize i.e. DllMain is a HANDLE / HINSTANCE, so it should stay `ptr`.
```suggestion
@ stdcall UserClientDllInitialize(ptr long ptr) DllMain
```

File: win32ss/user/user32/user32.spec (hunk 1 above)
Comment (tkreuzer) [reply win32ss/user/user32/user32.spec]: Fixed

File: win32ss/user/user32/user32.spec (hunk 2 above)
Comment (tkreuzer) [reply win32ss/user/user32/user32.spec]: fixed

**Review by TAN-Gaming — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

**Pull request merged**